documentation/src/main/asciidoc/topics/con_protostream_sci_implementations.adoc

```diff
@@ -0,0 +1,30 @@
+[id='protostream-sci-implementations_{context}']
+= Creating serialization context initializers
+
+A serialization context initializer lets you register the following with {brandname}:
+
+* Protobuf schema that describe user types.
+* Marshallers that provide serialization and deserialization capabilities.
+
+To create a serialization context initializer, you should:
+
+. Add ProtoStream annotations to your Java classes.
+. Use the ProtoStream processor that {brandname} provides to compile your `SerializationContextInitializer` implementation.
+
+.Manual implementations
```

Comment:
I don't think this should be in the opening `2.2` section. This is something we want to discourage and is for advanced use-cases, so we should put this much later in the chaper, probably at the end in 2.2.7.

Reply:
Also it is strongly not recommended to do so

Reply:
I've moved this to the end and made it clear that it is strongly recommended not to do this.